protocols.csv

```diff
@@ -35,7 +35,7 @@ code,	size,	name,	comment
 277,	0,	p2p-stardust,
 275,	0,	p2p-webrtc-star,
 276,	0,	p2p-webrtc-direct,
-280,	0,	webrtc, ICE-lite webrtc transport
-281, 	0,	webrtc-w3c, webrtc transport where connection establishment is according to w3c spec
+280,	0,	webrtc+sdp-munging, ICE-lite webrtc transport with SDP munging during connection establishment and without use of a STUN server
```

Comment:
```suggestion
280,	0,	webrtc-with-sdp-munging, ICE-lite webrtc transport with SDP munging during connection establishment and without use of a STUN server
```

AFAIK, we don't need to save chars for performance reasons so this is perhaps a tad easier to understand?